### DISCUSSION TITLE
Text content does not match server-rendered HTML.

### DISCUSSION THREAD

**Discussion by yashpareekk** [vercel/next.js#65273] [Help]
### Summary

Warning: Text content did not match. Server: "Products & Solutions" Client: "उत्पाद एवं समाधान"

I am getting this error whenever i change my language from English to Hindi

**Comment by icyJoseph** [comment]
Hi,

Are you perhaps doing a `toLowerCase`, or any other localization in your server side code? That won't be the same when running the same code in the client? 

For example: https://github.com/vercel/next.js/issues/46038
